services/Directory/data/Syntax.json

```diff
@@ -134,13 +134,9 @@
     "id": 31,
     "name": "URLRedirector"
   },
-  {
-    "id": 32,
```

Comment:
I _think_ the build error is now something tied to removing this syntax. Maybe there's another link still to `32` that also needs to be removed.